Readme.md

```diff
@@ -5,68 +5,36 @@
 - [Support](#support)
-- [Developing](#developing)
 - [Deploying](#deploying)
+- [Resources](#resources)
+- [License](#license)
 
 
 ## Installing
 
-To get up and running, check out our documentation at [https://segment.com/docs/plugins/magento](https://segment.com/docs/plugins/magento). Installation takes less than five minutes!
+Read the [installation instructions](./Install.md) for details on installing this plugin.
 
 
 ## Support
 
-If you run into issues, be sure to check out the [documentation](https://segment.com/docs/plugins/magento), and you can always reach out to our [support team](https://segment.com/support) for help!
-
-
-## Developing
-
-_Note: this section only applies if you are interested in contributing improvements to our Magento extension's source code. You don't need to read this to use the extension in your own store._
-
-TODO: We don't yet have detailed instructions for installing a local development version of Magento.
+Priority for support is granted to customers with an independent contract with Windsor Circle.
 
 
 ## Deploying
 
-_Note: this section only applies to Segment teammates, and details how we deploy new versions of the extension to the Magento Connect marketplace for end users of the extension to download._
-
-Before deploying, make sure to pull in the latest changes from the remote repository:
-
-    $ git pull origin master
-
-The first step after making your changes to the extension source is to tag a new release of the extension. To do that, first bump the version number in [`app/etc/modules/WindsorCircle_Analytics.xml`](app/code/community/WindsorCircle/Analytics/etc/config.xml).
-
-Then, update [`History.md`](History.md) with a summary of your changes.
-
-Once you've bumped the version and updated the changelog, you need to rebuild the distribution files in [`dist/`](dist/) by running the build script:
-
-    $ bin/build
-
-Then commit those changes, and tag the release (where `X.X.X` is the number of the newest version):
-
-    $ git commit -am 'X.X.X'
-    $ git tag 'X.X.X'
-    $ git push origin master
-    $ git push origin X.X.X
-
-Once you've done that, you'll need to log into our [Magento Connect]() account, and navigate to **Your Account > Developers > Manage Extensions** and click the **Edit** button on the **Analytics for Magento** extension. That will take you to the extension editing page. From there, click the **Versions** tab and then the **Add new version** button.
-
-Fill out the form for adding a new version, supplying the `X.X.X` version number as the **Release Notes Title** as well, and the list from [`History.md`](History.md) as the **Release Notes**. For the **Versions** check `1.7` and above. Then click the **Continue to upload** button.
-
-On the upload form page click **Choose File** and find the `analytics-magento-X.X.X` file on your computer from the [`dist/`](dist) directory, then hit **Upload and save**. 
-
-And you're finally done!
+TODO: This plugin might be deployable!
```

Comment:
Probably can just remove this section for now and open an issue to allow deployment to Magento Connect.  Or leave that issue in trello.  This isn't especially professional looking for now.  README is basically the face of the project